source/crud/crud.rst

```diff
@@ -189,7 +189,6 @@ Read
      * The number of documents to return per batch.
      *
      * This option is sent only if the caller explicitly provides a value. The default is to not send a value.
-     * For servers < 2.6, this option is ignored and should not be sent as aggregation cursors are not available.
```

Comment:
This likely should have been removed in https://github.com/mongodb/specifications/commit/7c5344205da658450228d2cd5b93f67f3c5c930e#diff-5efd2e4ce61f4ccda0ce031099303996 for [SPEC-836](jira.mongodb.org/browse/SPEC-836).